tests/parsers/sqlite_plugins/interface.py

```diff
@@ -78,7 +78,7 @@ def ParseMyTableRow(self, parser_mediator, query, row, **unused_kwargs):
 
     # If Python 2 is used field3 needs to be converted to a string
     # because it is a read-write buffer.
-    if sys.version_info[0] < 3:
+    if py2to3.PY_2 and field3:
```

Comment:
`and field3 is not None` because `''` should be converted. Do you plan to move this into interface.py (under plaso) ? If so add a TODO and link to the issue.

Reply:
Done,

Reply:
This is actually done now.